apps/condo/domains/billing/access/AllResidentBillingVirtualReceiptsService.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Generated by `createservice billing.AllResidentBillingVirtualReceiptsService --type queries`
+ */
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+async function canAllResidentBillingVirtualReceipts ({ authentication: { item: user } }) {
```

Comment:
Maybe allow only for residents?

Reply:
This is going to work exactly as allResidentBillingReceipt query